src/live-programming.rst

```diff
@@ -36,8 +36,132 @@ Direct Manipulation Programming
 Graphical User Interface Design
 -------------------------------
 
-Game Development
-----------------
+User Interfaces for Game Development
+====================================
+.. :Authors: - Lei Zhang
+
+Overview
+--------
+Starting with Spacewar in 1961, computer games have been around for over four and a half decades.
+Computer games are becoming increasingly important in lots of industries including software development, entertainment, education, military, and medicine.
```

Comment:
- "are becoming increasingly important in lots of" -> "are not just for fun. They are also being used for training and discovery in many"